tests/testthat/test-simulate-design.R

```diff
@@ -75,3 +75,15 @@ test_that("no estimates estimands declared", {
 
   expect_error(simulate_design(my_design_noestmand, sims = 2), "No estimates or estimands were declared, so design cannot be simulated.")
 })
+
+
+test_that("fan out IDs are correct", {
+  
+  skip_if_not_installed("DesignLibrary")
+  
+  sims <- c(3, 1, 2, 1, 1, 2)
+  design <- DesignLibrary::simple_two_arm_designer(rho = 0)
+  
+  simulate_design(design, sims = sims)[, c(4, 7, 15:19)]
```

Comment:
this needs an expect_something somewhere, ow testthat might skip it.

Reply:
done, thx